### PR TITLE
update dependencies and clean up code so build succeeds

### PR DESCRIPTION
**Pull request by roseja**
I tested that this PR would build locally with `yarn build` so that deploys can be unblocked.

Screen Shot 2022-02-11 at 3 28 58 AM

### CODE REVIEW

File: pages/bed-index-planner.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default function BedIndex({
           <Text mt="0 !important">
             {moment(activePrice.timestamp).format('DD MMMM, YYYY')}
           </Text>
-          <Box spacing={3} w="full" h={260}>
+          <Box w="full" h={260}>
```
Comment (roseja) [inline pages/bed-index-planner.tsx:251]: This was causing an error, but I want to make sure the design isn't drastically changed by removing spacing. This might need another update.

Comment (geositta) [reply pages/bed-index-planner.tsx:251]: My guess is that this prop was not actually doing anything, given that this is not a chakra-ui expected prop, nor is it any CSS attribute. The closest thing I suppose is "letter-spacing."

Are you able to post screenshots in the PR of before and after the change removing the spacing?

I also think that using a width of "full" is not conventional, and I wonder if that width is even doing anything.
https://developer.mozilla.org/en-US/docs/Web/CSS/width

In summary, the existing code is wrong on at least 2 of the three attribute props. I don't think your change is introducing any sort of regression, but please if you would just clarify what you see locally.

Comment (CompositeFellow) [reply pages/bed-index-planner.tsx:251]: > My guess is that this prop was not actually doing anything, given that this is not a chakra-ui expected prop, nor is it any CSS attribute. The closest thing I suppose is "letter-spacing."
> 
Spacing is special Chakra prop that allows you to quickly and evenly space.  https://chakra-ui.com/docs/layout/wrap#change-the-spacing for some quick example

> I also think that using a width of "full" is not conventional, and I wonder if that width is even doing anything. https://developer.mozilla.org/en-US/docs/Web/CSS/width
> 
Full is also a special Chakra UI thing that sets the width of the element to full screen size.

I'm having trouble with my local Ghost today, but Ill take a look as soon as I resolve the issue

Comment (geositta) [reply pages/bed-index-planner.tsx:251]: The spacing prop looks like it is expected on Wrap or SimpleGrid components but possibly the issue is that it is not expected on Box component.

Comment (roseja) [reply pages/bed-index-planner.tsx:251]: There were a few dependency issues, so I upgraded chakra `"@chakra-ui/icons": "^1.1.5", "@chakra-ui/react": "^1.8.1"`
I do believe spacing was a valid attribute in the older versions, but has now been deprecated. Looking at these screenshots, I don't see a noticeable different. The update was around the box/price chart. https://www.bankless.community/bed-index-planner

-----------------------------------------------------------------------------------------------
### Current Production with spacing
current-prod

-----------------------------------------------------------------------------------------------
### Preview with out spacing
preview-site

**Comment by roseja** [comment]
If this looks good to everyone can we approve/merge this in and see if that fixes the production deploy pipeline?

**Review by geositta — APPROVED** [review]

**Review by geositta — APPROVED** [review]

**Comment by geositta** [comment]
I approved the PR but I don't have write access to repo so that is why the merge is still blocked.

**Comment by roseja** [comment]
@KenMan79 do you have write access? Not sure if we can get this approved and verify the analytics are deployed to production.

**Review by singularityhacker — APPROVED** [review]
